src/pybind/mgr/cephadm/module.py

```diff
@@ -1135,7 +1135,10 @@ def _remote_connection(self,
             self._reset_con(host)
 
             user = self.ssh_user if self.mode == 'root' else 'cephadm'
-            msg = f'''Failed to connect to {host} ({addr}).
+            if str(e).startswith("Can't communicate"):
+                msg = str(e)
+            else:
+                msg = f'''Failed to connect to {host} ({addr}).
```

Comment:
does it make sense to also add the error message?

```suggestion
                msg = f'''Failed to connect to {host} ({addr}): {e}.
```